sszgen/main.go

```diff
@@ -315,56 +315,28 @@ func (e *env) generateEncodings() (map[string]string, error) {
 	return outs, nil
 }
 
-var errorFunctions = map[string]string{
```

Comment:
```
100644 → 100755
```

Should this stay with file permission 644? Why does it need to be executable?

Reply:
Good catch. Not sure why but the permissions of the files have changed.